scripts/utils/env-variable-value.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo_env_variable_value() {
```

Comment:
This seems to me a bit fragile as an alternative to sourcing the `.env` file. When sourcing files comments are allowed but this script does not handle comments.

Reply:
I prefer the global env variables over the .env file's variables, that's why I wrote this script.
Nevertheless your comment is justified, I make some adjustions